tests/Behat/features/add-block-element-to-data-object.feature

```diff
@@ -5,7 +5,8 @@ Feature: Add elements in the CMS DataObject
   So that I can use multiple elements in a Dataobject
 
   Background:
-    Given I am logged in with "ADMIN" permissions
+    Given the "group" "EDITOR group" has permissions "Access to 'Pages' section" and "ELEMENTAL_DATAOBJECT_EDIT"
```

Comment:
```suggestion
    Given the "group" "EDITOR" has permissions "Access to 'Pages' section" and "ELEMENTAL_DATAOBJECT_EDIT"
```
There should be no need to add "group" to the end of the group name. Obviously the rest of these will need to change as well.